flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointFailureReason.java

```diff
@@ -81,6 +81,8 @@
 
     FINALIZE_CHECKPOINT_FAILURE(false, "Failure to finalize checkpoint."),
 
+    IO_EXCEPTION(false, "Trigger checkpoint failure."),
```

Comment:
Please fix the description. right now it is just copy-paste of `TRIGGER_CHECKPOINT_FAILURE`.

Reply:
Sorry for that and I have fixed this.